### ISSUE TITLE
Staging Review - Accessibility Feedback - Design System Team - OMB info component

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#48546]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
-

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**
Still can't manage to get a screen reader to announce the contents of the modal. I've tested with NVDA and JAWS on desktop and VoiceOver on iOS. The regular `va-modal` component doesn't have any issues, so there's something subtle happening with how it's being used in the `va-omb-info` component --- or something weird is going wrong on my end.

Here's a [quick recording](https://user-images.githubusercontent.com/6844775/197029568-3445b105-ce85-4795-8fb6-27b5149dcf98.mp4) of me fussing with NVDA on both the `va-modal` demo in Storybook and the `va-omb-info` demo.

Let me know if you want to chat about this.

**Should:**

**Consider:**

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by jamigibbs** [comment]
I suspect that the issue is originating in `va-modal` where it's adding `aria-hidden=true` to all elements outside of the modal for focus trapping: https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-modal/va-modal.tsx#L293-L295

When I remove this functionality, the screenreader functions for me. Perhaps checking in `va-omb-info` that aria-hidden is not being added to elements there would resolve this issue.
